src/components/genre-selector/GenreSelector.tsx

```diff
@@ -18,7 +18,7 @@ const GenreSelector: FC<GenreSelectorProps> = ({ genres, currentGenre, onSelecte
   return (
     <ul className={styles.selector}>
       {genres.map(item => (
-        <li key={item}>
+        <li key={item} data-cy="genre-link">
```

Comment:
standard practice: use `data-testid` attribute

Reply:
and with react-testing-library - ***byTestId** https://testing-library.com/docs/queries/bytestid/